testing_integration/dual_proxy_test.sh

```
@@ -0,0 +1,157 @@
+    COMPARISON_RESULT=$?
+else
+    echo -e "${YELLOW}⚠️  Python3 not found, skipping automated comparison${NC}"
+    echo -e "${BLUE}Manual comparison:${NC}"
+    echo -e "  diff /tmp/passthrough_output.txt /tmp/cerebras_output.txt"
+    COMPARISON_RESULT=0
+fi
+
+# Cleanup: Stop Cerebras proxy
+echo -e "\n${BLUE}Cleanup: Stop Cerebras proxy${NC}"
+if lsof -ti:10001 >/dev/null 2>&1; then
+    kill $(lsof -ti:10001) 2>/dev/null || true
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)